src/algebra/group/to_additive.lean

```diff
@@ -57,6 +58,29 @@ private meta def aux_attr : user_attribute (name_map name) name :=
 
 end performance_hack
 
+section extra_attributes
+
+setup_tactic_parser
+
+/-- An attribute that tells `@[to_additive]` that certain arguments of this definition are not
+  involved when using `@[to_additive]`.
+  This helps the heuristic of `@[to_additive]` by also transforming definitions if `ℕ` or another
+  fixed type occurs as one of these arguments. -/
```

Comment:
IIRC, indenting continuation lines in a docstring causes them to appear as code.

I'd also include the name as seen by the user.
```suggestion
/--
The attribute `to_additive_ignore_args` tells `@[to_additive]` that certain arguments of
this definition are not involved when using `@[to_additive]`.
This helps the heuristic of `@[to_additive]` by also transforming definitions if `ℕ` or another
fixed type occurs as one of these arguments.
-/
```